repose-aggregator/components/filters/rate-limiting/src/test/java/org/openrepose/filters/ratelimiting/RateLimitingHandlerTest.java

```diff
@@ -81,7 +82,7 @@ public void shouldReturnUnauthorizedWhenUserInformationIsMissing() {
 
         @Before
         public void setup() {
-            List<String> headerNames = new ArrayList<String>();
+            final List<String> headerNames = new ArrayList<>();
```

Comment:
Since the `headerNames` variable is now accessed directly in the anonymous `Answer` class it has to be declared `final`.